torchvision/utils.py

```diff
@@ -575,3 +549,63 @@ def _make_ntuple(x: Any, n: int) -> Tuple[Any, ...]:
     if isinstance(x, collections.abc.Iterable):
         return tuple(x)
     return tuple(repeat(x, n))
+
+
+def _parse_colors(
+    colors: Optional[Union[List[Union[str, Tuple[int, int, int]]], str, Tuple[int, int, int], None]],
```

Comment:
1. Reordering for better legibility
2. `Optional[Union[..., None]] == Union[..., None]`

```suggestion
    colors: Union[None, str, Tuple[int, int, int], List[Union[str, Tuple[int, int, int]]]],
```